dhall-json/src/Dhall/JSON.hs

```diff
@@ -61,7 +61,7 @@
     Dhall @Optional@ values translate to @null@ if absent and the unwrapped
     value otherwise:
 
-> $ dhall-to-json <<< 'None Natural'
+> $ dhall-to-json --preserveNull <<< 'None Natural'
```

Comment:
The flag shouldn't be necessary here, `omitNull` doesn't change a single `Null`.